build.gradle

```diff
@@ -245,17 +245,18 @@ project('segmentstore:storage:impl') {
 
 project ('bindings') {
     dependencies {
-        // https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-common
-        compile group: 'org.apache.hadoop', name: 'hadoop-common', version: hadoopVersion, withoutLogger
+        // For filesystem
+        compile group: 'org.apache.jclouds.provider', name: 'google-cloud-storage', version: googleCloudStorageProviderVersion
```

Comment:
Why do we add this dependency?

Reply:
It is needed for the `ExtendedS3StorageTest` when creating an S3Proxy implementation in https://github.com/pravega/pravega/blob/master/bindings/src/test/java/io/pravega/storage/extendeds3/S3ProxyImpl.java#L60

If not present, the test fail and throw `java.util.ServiceConfigurationError: org.jclouds.providers.ProviderMetadata: Provider org.jclouds.googlecloudstorage.GoogleCloudStorageProviderMetadata could not be instantiated`.

However, I'll change the dependency from `compile` to `testCompile`.